README.md

```diff
@@ -18,6 +18,62 @@ dev_appserver .
 
 See [CONTRIBUTING.md](/CONTRIBUTING.md) for more information on local development.
 
+## Using the data
+
+All test result data is public. There are two types of gzipped JSON data files we store.
+
+### Test run summary files
+
+These are of the pattern: `{sha[0:10]}/{platform_id}-summary.json.gz`
+
+- `sha[0:10]` - the first 10 characters of the WPT SHA that run was tested against
```

Comment:
I'm not sure if it's clearer, but I'd write "WPT commit hash" instead of "WPT SHA". And perhaps we could use a colon instead of a hyphen/minus (here and below).